docs/amd_drug_metabolite.rst

```diff
@@ -0,0 +1,421 @@
++----------------------+---------------------------------------------------------------------------------------------+
+
+COVSearch
+~~~~~~~~~
+
+The settings that the AMD tool uses for this subtool can be seen in the table below. The effects are extracted from the
+search space.
+
++---------------+----------------------------------------------------------------------------------------------------+
+| Argument      | Setting                                                                                            |
++===============+====================================================================================================+
+| effects       | ``search_space`` (As defined in :ref:`AMD options<amd_drug_metabolite_args>`)                      |
++---------------+----------------------------------------------------------------------------------------------------+
+| p_forward     | ``0.05``                                                                                           |
++---------------+----------------------------------------------------------------------------------------------------+
+| p_backward    | ``0.01``                                                                                           |
++---------------+----------------------------------------------------------------------------------------------------+
+| max_steps     | ``-1``                                                                                             |
++---------------+----------------------------------------------------------------------------------------------------+
+| algorithm     | ``'scm-forward-then-backward'``                                                                    |
++---------------+----------------------------------------------------------------------------------------------------+
+
+If no search space for this tool is given, the following default will be used:
+
+.. code-block::
+
+    COVARIATE?(@IIV, @CONTINUOUS, exp, *)
+    COVARIATE?(@IIV, @CATEGORICAL, cat, *)
+
+
+
+.. graphviz::
+
+    digraph BST {
+            node [fontname="Arial",shape="rect"];
+            rankdir="LR";
+            base [label="Input", shape="oval"]
+            s0 [label="mechanistic covariates"]
+            s1 [label="exploratory covariates"]
+
+            base -> s0
+            s0 -> s1
+        }
+
+
+
+**Mechanisitic covariates**
+
+If any mechanistic covariates have been given as input to the AMD tool, the specified covariate effects for these
+covariates is run in a separate initial covsearch run when adding covariates. These covariate effects are extracted
+from the given search space
+
+**Exploratory covariates**
+
+The covariate effects remaining in the search space after having run potentially both structural and mechanistic covariates
+are now run in an exploratory search.
+
+**Examples**
+
+.. code-block::
+
+    mechanistic_covariates = [AGE, (CL,WGT)]
```

Comment:
Yes it is an AMD argument. But the argument need to be specified in the example as well (otherwise i cannot show how it affects the search space. but i can move it outside of the code block